client/src/templates/Introduction/components/block.tsx

```diff
@@ -193,12 +193,23 @@ export class Block extends Component<BlockProps> {
               }}
             >
               <Caret />
-              <h4 className='course-title'>
-                {`${isExpanded ? collapseText : expandText}`}
-              </h4>
+              <div className='course-title'>
+                {`${isExpanded ? collapseText : expandText}`}{' '}
+                <span className='sr-only'>{blockTitle}</span>
+              </div>
               <div className='map-title-completed course-title'>
                 {this.renderCheckMark(isBlockCompleted)}
-                <span className='map-completed-count'>{`${completedCount}/${challengesWithCompleted.length}`}</span>
+                <span
+                  aria-hidden='true'
+                  className='map-completed-count'
+                >{`${completedCount}/${challengesWithCompleted.length}`}</span>
+                <span className='sr-only'>
+                  ,{' '}
+                  {t('learn.challenges-completed', {
+                    completedCount,
+                    totalChallenges: challengesWithCompleted.length
+                  })}
```

Comment:
This looks great. I was wondering if you have come across a solution for languages with non-english numbers @moT01.

Reply:
I haven't really ever looked into it. A quick search, I found two packages that look okay: https://www.npmjs.com/package/numbo & https://www.npmjs.com/package/n2words. We would probably want to pair up with native speakers to see if converting the numbers in the translations still makes sense. We could just add numbers to the translation object and put it on crowdin - that's probably our best option. I don't think we need to have that many numbers - the longest projects are maybe 150 steps. Then there's also the other miscellaneous numbers across the site, which can already be translated... but they aren't getting translated:
![Screen Shot 2022-06-14 at 7 57 50 AM](https://user-images.githubusercontent.com/20648924/173583272-decd1ab5-49f8-42c9-bd85-1bd3cca04f6f.png)
hmm - don't know. Maybe it's common to use English numbers in Chinese writing or something.

Reply:
To me, a native Japanese speaker and understands a little bit of Chinese, those packages don't look useful here.

At least for Japanese, having a string with interpolation of numbers is the best option. 
We can just use Arabic numbers in Japanese (and it's more natural), but, often the order of the words are different.

Such as the current string: `{{completedCount}} of {{totalChallenges}} challenges completed` is fine.
It can be translated into something like: `{{totalChallenges}}件中{{completedCount}}件完了` and eventually rendered such as `150件中10件完了`

Reply:
Thanks for the input here. I think any changes like this would be outside the scope of the PR. If we want to add this, we should make a new issue. I am going to resolve this - feel free to unresolve if you want to continue the conversation.